hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -333,8 +334,7 @@ forAllClose ::
   (UTxO -> Tx -> property) ->
   Property
 forAllClose action = do
-  -- FIXME: we should not hardcode number of parties but generate it within bounds
```

Comment:
It's still hard-coded. Just a different value. What this FIXME says is that we should generate between `1..maximumNumberOfParties`

Reply:
Same holds for these other generators. They should try most possible combinations / be "as arbitrary as possible".